client/src/components/ItemCard/ItemCardMedia.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import { useTheme } from "@material-ui/core/styles";
+
+function ItemCardMedia({
+  media,
+  height,
+}: {
+  media: string | null | undefined;
```

Comment:
media should not be undefined IMO, either it is provided or it defaults to a generic image/text

Reply:
moved checking for media definition into `ItemCard` instead and renders (static) placeholder image if `item.media` is not defined.
We would likely move this to a server call to retrieve default set of resources based on item name? perhaps add a label type field and serve up images based on this label type (food, drink, utensils... etc?)

Reply:
like for starters it can even be 1 generic thing for every item